### PR TITLE
Update figures

### LINKED ISSUE #74: Figures outdated
## Description

Figures in the tutorials are outdated, e.g. the overview and kernel choice figures in the getting-started tutorial.

### PR DESCRIPTION
**Pull request by WeilerP**
## Changes

- Update overview figures
- Update kernel choice figure and add dark mode version
- Update the getting started tutorial to support dark mode for kernel choice figure

## Related issues

Closes #74.

### CODE REVIEW

File: tutorials/_static/img/100_cellrank_kernels.png
Comment (michalk8) [inline tutorials/_static/img/100_cellrank_kernels.png:1]: I'd suggest having the `light_mode` postfix for consistency.

File: tutorials/_static/img/100_cellrank_overview_dark_mode.png
Comment (michalk8) [inline tutorials/_static/img/100_cellrank_overview_dark_mode.png:1]: I think the dark mode PNG is again bigger than the light mode (3352x1708 vs 1846x941).

File: tutorials/general/100_getting_started.ipynb [hunk 1]
```diff
@@ -557,9 +557,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    ":::{image} ../_static/img/100_cellrank_kernels_mode.png\n",
```
Comment (michalk8) [inline tutorials/general/100_getting_started.ipynb:560]: Think this should be `100_cellrank_kernels_dark_mode.png`.

File: tutorials/_static/img/100_cellrank_overview.png
Comment (michalk8) [inline tutorials/_static/img/100_cellrank_overview.png:1]: Would also consider renaming to `100_cellrank_overview_light_mode.png` for consistency + updating the links in the notebook(s).

File: tutorials/general/100_getting_started.ipynb [hunk 2]
```diff
@@ -557,9 +557,18 @@
...
+    ":class: only-dark\n",
+    ":width: 800px\n",
+    ":::\n",
+    "\n",
+    ":::{figure-md} fig-structure\n",
```
Comment (michalk8) [inline tutorials/general/100_getting_started.ipynb:565]: Please remove the `:::{figure-md} fig-structure` part - there are now 2 images being rendered.

**Review by michalk8 — CHANGES_REQUESTED** [review]
Minor fixes, then LGTM! Thanks @WeilerP !

File: tutorials/_static/img/100_cellrank_overview_dark_mode.png
Comment (WeilerP) [reply tutorials/_static/img/100_cellrank_overview_dark_mode.png:1]: Yes, strange. I fixed it.

**Review by michalk8 — APPROVED** [review]
LGTM, thanks a lot!

**Review by Marius1311 — APPROVED** [review]
Looks great!

**Pull request merged**